algorithms/Cargo.toml

```diff
@@ -129,7 +129,7 @@ features = [ "derive" ]
 
 [dependencies.sha2]
 version = "0.10"
-default-features = false
+default-features = true
 
```

Comment:
@howardwu internally in our SNARK, we use a hash function to derive a digest of a circuit. For this, should we use `sha2`, or another hash function (e.g. `blake2`)?

Reply:
Is Poseidon possible? My suggestion is to use blake2s otherwise. We currently don't use sha in our protocol.

Reply:
Poseidon would be much too slow. @vicsn changed it to use blake2s, but as he noted, this adds a dependency to `algorithms`.